package-lock.json

```diff
@@ -4213,7 +4232,7 @@
       }
     },
     "fxa-dev-launcher": {
-      "version": "github:vladikoff/fxa-dev-launcher#f5a01841fb6f3ca3fabf711219ed1cf94c8d63e6",
+      "version": "github:vladikoff/fxa-dev-launcher#f50f8111aa5470d637d1db96050556b5f1b12b2e",
```

Comment:
Hmm, only thing here that made me raise my eyebrows was the change in commit hash here. But, [we don't pin this to a particular commit in `package.json`](https://github.com/mozilla/fxa/blob/master/package.json#L29), and we probably want [that change](https://github.com/vladikoff/fxa-dev-launcher/commit/f5a01841fb6f3ca3fabf711219ed1cf94c8d63e6) anyway...

Reply:
@lmorchard ah yeah, I had in my PR that I'd like to confirm that was correct since I wasn't expecting that to show. 🤔 From your last link it seems we do want that, though.